devel/cctools/Portfile

```diff
@@ -88,45 +86,79 @@ if {![some_llvm_variant_set]} {
     #     default_variants +llvm33
     # }
 
-    if {![some_llvm_variant_set] && ${os.major} >= 10} {
+    if {[vercmp $xcodeversion 10.0] >= 0} {
+        default_variants +xcode
+    } elseif {![some_llvm_variant_set] && ${os.major} >= 10} {
         default_variants +llvm50
     }
+
+}
+
+variant xcode description "Use Xcode supplied toolkit" { }
+if {[variant_isset xcode]} {
+   patch { }
```

Comment:
You should basically never override the patch phase to disable it; instead, clear `patchfiles`.

Reply:
Whats the syntax to do this ?

Reply:
Don't worry, eventually guess right ;)